web/themes/custom/novel/templates/views/eventinstance--stacked-event.html.twig

```diff
@@ -0,0 +1,8 @@
+
+<a href="{{ eventinstance_url }}" class="event-list-item-stacked arrow__hover--right-small">
+  <time class="event-list-item-stacked__content" datetime="{{ datetime_attribute }}">
+    <span class="event-list-item-stacked__date">{{ content.date }}</span>
+    <span class="event-list-item-stacked__time">{{ start_time }} - {{ end_time }}</span>
+  </time>
+  {{ source(directory ~ '/assets/dpl-design-system/icons/arrow-ui/icon-arrow-ui-small-right.svg') }}
```

Comment:
Please use the `baseIconPath` variable like [elsewhere](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/f7f7dce01cfa4a4d4dd10f50e1b7d9cfca0ed009/web/themes/custom/novel/templates/components/nav-spot.html.twig#L28-L28).